src/results.js

```diff
@@ -201,7 +179,7 @@ class Results extends React.Component {
           </div>
           {preview &&
             <div className='col-sm-8' key={previewUrl}>
-              <Preview url={previewUrl} />
+              <DocPage docUrl={previewUrl} />
```

Comment:
`DocPage` was intended to be used only from `doc.html`, in case we need some logic specific to that page, that doesn't belong in the preview. I think we should create a new component, `Document`, that's used from `Results` and `DocPage`, and does all of the work of rendering a document.